nav2_amcl/src/amcl_node.cpp

```diff
@@ -213,6 +213,9 @@ AmclNode::AmclNode()
     "Requires that AMCL is provided an initial pose either via topic or initial_pose* parameter "
     "(with parameter set_initial_pose: true) when reset. Otherwise, by default AMCL will use the"
     "last known pose to initialize");
+
+  add_parameter("topic", rclcpp::ParameterValue("scan"));
```

Comment:
Add description to this

Reply:
also "topic" is a poor name, we need to specify what this topic is for / composed of. How about `scan_topic`?

Reply:
I tried to stick with the same convention as the costmap layers but I guess the name spacing makes a difference. Ill go ahead and change it.